### PR TITLE
fix: info toast long title

### PR DESCRIPTION
**Pull request by Viraj-10**
## Summary
Info Icon and close button was going out of the screen for android and ios device when title of the info toast is long.

### CODE REVIEW

File: src/components/composites/Toast/Toast.tsx [hunk 1]
```diff
@@ -271,15 +271,15 @@ export const ToastProvider = ({ children }: { children: any }) => {
           <Alert
             maxWidth="100%"
             alignSelf="center"
+            flexDirection="row"
             status={status ?? 'info'}
             variant={variant as any}
             accessibilityLiveRegion={accessibilityLiveRegion}
             {...rest}
           >
-            <VStack space={1} flexShrink={1} w="100%">
```
Comment (rayan1810) [inline src/components/composites/Toast/Toast.tsx:279]: No need to remove w="100%"

File: src/components/composites/Toast/Toast.tsx [hunk 2]
```diff
@@ -271,15 +271,15 @@ export const ToastProvider = ({ children }: { children: any }) => {
...
+            <VStack space={1} flexShrink={1}>
               <HStack
                 flexShrink={1}
-                space={2}
```
Comment (rayan1810) [inline src/components/composites/Toast/Toast.tsx:282]: Why did we removed the spacing here?

**Review by rayan1810 — CHANGES_REQUESTED** [review]

File: src/components/composites/Toast/Toast.tsx (hunk 2 above)
Comment (Viraj-10) [reply src/components/composites/Toast/Toast.tsx:282]: it was creating a issue with info icon and close button. Both were not having same distance from text when title is long on IOS and Android. small UI change suggested by Himanshu.

File: src/components/composites/Toast/Toast.tsx (hunk 1 above)
Comment (Viraj-10) [reply src/components/composites/Toast/Toast.tsx:279]: added w="100%"

**Review by rayan1810 — APPROVED** [review]

**Pull request merged**